recipes/pymvpa2/meta.yaml

```diff
@@ -0,0 +1,82 @@
+# Use
+# python -c 'from jinja2 import Template; print(Template(open("meta.yaml").read()).render())'
+# to see templating interpolated
+{% set name = "pymvpa2" %}
+{% set version = "2.6.1" %}
+{% set sha256 = "206fa302fc6292831c50d6ffe9b1a95fc048380b0701c140eb7300dcd017052a" %}
+# Common dependencies between run and build (Yarik hates duplication)
+{% set common_deps = "
+    - python
+    - numpy \
+" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py>3]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  # TODO: entry_points -- well, we have scripts
+
+requirements:
```

Comment:
It looks like this package is building against numpy. Please see:

https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
done, thanks. mimiced after scikit-learn which we use